Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -1078,9 +1087,7 @@ extension TabDisplayManager: TabManagerDelegate {
         /// collectionView is not visible the dataSource section/items differs from the actions to be perform
         /// which causes the crash
         collectionView.numberOfItems(inSection: 0)
-        collectionView.performBatchUpdates({ [weak self] in
-            // Baseline animation speed is 1.0, which is too slow, this (odd) code sets it to 3x
-            self?.collectionView.forFirstBaselineLayout.layer.speed = 3.0
```

Comment:
Removing this is intentional?

Reply:
Yes `forFirstBaselineLayout` is a property F`or views with multiple rows of text` as per Apple's documentation so I couldn't find anything justifying this line of code